docs/README.md

```diff
@@ -0,0 +1,97 @@
+
+## Table of Contents
+
+- [Features](#features)
+- [Supported Devices / OS](#supported-devices--os)
+- [Get Started](#get-started)
+  - [Install](#install)
+  - [Run](#run)
+  - [More Options](#more-options)
+- [Screenshots](#screenshots)
+  - [TV](#tv)
+  - [Web interface](#web-interface)
+- [Troubleshooting](#troubleshooting)
+- [Install from Repo](#install-from-repository-legacy)
+
+## Features
+
+| **Feature**                            | **Description**                                      |
+|----------------------------------------|------------------------------------------------------|
+| Web Interface                          | Multiple users can queue tracks                     |
+| Player/Splash Screen                   | Connection QR code and "Next up" display             |
+| Searching/Browsing                     | Browse a local song library                         |
+| Adding New Songs                       | Add new songs from Youtube                          |
+| mp3 + cdg Support                      | Includes compressed .zip bundles                    |
+| Playback Controls                      | Pause, Skip, Restart, and volume control            |
+| File Name Editing                      | Advanced editing of downloaded file names           |
+| Queue Management                       | Manage the song queue                               |
+| Key Change / Pitch Shifting            | Adjust the pitch of songs                           |
+| Admin Mode                             | Lock down features with admin mode                  |
+
+## Supported Devices / OS
+
+| **Device/OS**                         | **Details**                                                                                   |
+|---------------------------------------|-----------------------------------------------------------------------------------------------|
+| Raspberry Pi 3 and above              | Suitable, but anything below Pi 3 will likely be too slow.                                    |
+| Raspberry Pi Desktop OS (headed)      | Should be used if running with a display, as it requires a browser.                           |
+| 32-bit Bullseye OS (recommended for Pi 3) | Recommended OS for Pi 3.                                                                      |
+| 64-bit Bookworm OS (recommended for Pi 4 and above) | Fine for Pi 4 and above.                                                             |
+| Raspberry Pi 3 High-Res Video Playback | Pi 3 might struggle with high-res video playback, but overclocking can help.                  |
+| Disable Screen Blanking               | Disable "screen blanking" in `raspi-config` to prevent the display from turning off when idle.|
+| Modern Mac, PCs, and Linux            | Works fine on modern devices.                                                                 |
```

Comment:
I'm not sure this column designation makes sense. The first 6 items are notes about raspberry pi.

Reply:
It was just an attempt at making a table instead of just having text. It's all up to you